include/lsst/jointcal/ConstrainedPolyModel.h

```diff
@@ -24,7 +24,10 @@ namespace jointcal {
 
 class CcdImageList;
 
-//! This is the model used to fit mappings as the combination of a transformation depending on the chip number (instrument model) and a transformation per shoot (anamorphism). The two-transformation Mapping required for this model is TwoTransfoMapping.
+//! This is the model used to fit mappings as the combination of a
+//! transformation depending on the chip number (instrument model) and a
+//! transformation per visit (anamorphism). The two-transformation Mapping
+//! required for this model is TwoTransfoMapping.
 /*! This modeling of distortions is meant for set of images from a single mosaic imager. */
```

Comment:
I'll clean up some of these now, and see if there's a better way to do more of them later.